src/views/PokemonList.js

```diff
@@ -0,0 +1,15 @@
+import React, { useEffect, useState } from 'react';
+import { fetchPokemon } from '../services/pokemon';
+
+export default function PokemonList({ pokemon }) {
+  return (
+    <>
+      <div>PokemonList</div>
+      <ul>
+        {pokemon.map((poke) => (
+          <li key={poke.id}>{poke.pokemon} </li>
+        ))}
+      </ul>
+    </>
+  );
```

Comment:
This file can actually go into our component folder since it does not utilize state.